### ISSUE TITLE
drop php v5.6 ??

### ISSUE THREAD

**Issue by lowtower** [lowtower/zend-expressive2-tutorial#5]
Should we drop php v5.6 support?

The `zend-expressive2-skeleton` still supports php v5.6, but most `zend-...` packages do not.
Latest, when implementing `zend-expressive-navigation` php v5.6 will have to be dropped.

Question:
Should we drop php v5.6 in the tutorial right from the start?

**Comment by froschdesign** [comment]
@lowtower 
> Should we drop php v5.6 support?

Yes! PHP 7.2 is around the corner and the active support for 5.6 ended 9 months ago.

http://php.net/supported-versions.php

**Comment by lowtower** [comment]
Should the [skeleton](https://github.com/zendframework/zend-expressive-skeleton) be updated first as a new base for part1 or should I make changes to part2 earliest?
